cohortextractor/study_definition.py

```diff
@@ -428,7 +453,7 @@ def check_date_expectations_defined(self, colname, kwargs):
 
 
 def merge(dict1, dict2):
-    """ Return a new dictionary by merging two dictionaries recursively. """
```

Comment:
this was `black` not me!

Reply:
Ah, it looks like this is new behaviour from black and you must be running a different version from the one in `requirements.txt`. That's fine though, it's a sensible enough change and we should be using a more recent version of black anyway.